tensorflow/python/ops/signal/fft_ops.py

```diff
@@ -324,6 +325,83 @@ def _grad(op, grad):
 
   return _grad
 
+@tf_export("signal.fftshift")
```

Comment:
Please do

blaze run tensorflow/tools/api/tests:api_compatibility_test -- --update_goldens=True 

this will make the API compatibility tests pass (they're currently failing)

Reply:
sir, i am running test on google collab. as in my machine it taking lots of time.

Reply:
Yes but you need the output files of the test to be added to this PR